bindings/wasm/examples/worker.test.ts

```diff
@@ -55,71 +55,71 @@ afterEach(async () => {await new Promise(resolve => setTimeout(resolve, 500))})
 suite('Examples', () => {
   test('Intro', async () => {
     const result = await runExample('Intro');
-    expect(result.genus).to.equal(5, 'Genus');
-    expect(result.volume).to.be.closeTo(203164, 1, 'Volume');
-    expect(result.surfaceArea).to.be.closeTo(62046, 1, 'Surface Area');
+    expect(result?.genus).to.equal(5, 'Genus');
```

Comment:
Why all the ?s - can result actually be null?

Reply:
Yes: https://github.com/elalish/manifold/blob/209350fd1ee35749fecf2f6bd58a3e96cf8f0291/bindings/wasm/examples/worker.ts#L732